server/routerlicious/.gitignore

```diff
@@ -15,5 +15,5 @@ package-deps.json
 # Temporary api-extractor build artifacts
 _api-extractor-temp/**
 
-# Lerna Monorepo: Individual packages should not have package-locks
+# Monorepo: Individual packages should not have package-locks
 packages/*/package-lock.json
```

Comment:
No package should have apackage-lock.json since we use pnpm which uses pnpm-lock.yml. Can we just remove this line entirely?

Reply:
Also, even when we used learna this seems wrong as it ignored files that should not exist: we should not have lock files of any kind inside the mono repo, not even unversioned/locally right?

Reply:
Wow, our repo stil has ~34 references to package-lock.json. I guess cleaning those up is a separate thing.

Reply:
No, we shouldn't, and we have [several]() [policies]() to prevent them. The ignore entry is definitely superfluous/wrong. I think we should do a pass on our ignores and clean them up, and ordinarily I would postpone this sort of change for such a broader cleanup change. But... I'm right here, might as well make it better right now.